zeeguu_api/app.py

```diff
@@ -25,7 +25,7 @@
 
 try:
     import dashboard
-    dashboard.config.from_file('/home/mircea/zee/http/api/dashboard.cfg')
+    dashboard.config.from_file(None)
```

Comment:
this is misleading... 
maybe implement `from` in such a way that you can call it either 
    dash.config.from(file='/home/thijs/...')
or 
   dash.config.from(envvar='DASHBOARD_CONFIG')

that would be so much more readable :)

Reply:
Yes, I can definitely do this.